README.md

```diff
@@ -1,124 +1,32 @@
-
-```
-Arduino Firmware Uploader (arduino-fwuploader).
-
-Usage:
-  arduino-fwuploader [command]
-
-Examples:
-  ./arduino-fwuploader <command> [flags...]
-
-Available Commands:
-  certificates Commands to operate on certificates.
-  firmware     Commands to operate on firmwares.
-  help         Help about any command
-  version      Shows version number of arduino-fwuploader.
-
-Flags:
-      --format string       The output format, can be {text|json}. (default "text")
-  -h, --help                help for arduino-fwuploader
-      --log-file string     Path to the file where logs will be written
-      --log-format string   The output format for the logs, can be {text|json}.
-      --log-level string    Messages with this level and above will be logged. Valid levels are: trace, debug, info, warn, error, fatal, panic (default "info")
-  -v, --verbose             Print the logs on the standard output.
-
-Use "arduino-fwuploader [command] --help" for more information about a command.
-```
-
-## How to build the tools from source file
-
-To build we use [task](https://taskfile.dev/) for simplicity. From the sources root directory run:
-
-```
-task dist:<OS>_<ARCH>
-```
-
-Where <OS> could be one of: `macOS`,`Windows`,`Linux`. And <ARCH>: `32bit`, `64bit`, `ARM` or `ARM64`
-
-This will create the `arduino-fwuploader` executable.
+1. [Install] the Arduino Firmware Uploader
+2. Follow the [Usage] guide to check out what the tool can do
+3. Browse the [Commands reference] to see all the available commands
 
 ## Security
 
-If you think you found a vulnerability or other security-related bug in this project, please read our [security
-policy][security-policy] and report the bug to our Security Team 🛡️ Thank you!
+If you think you found a vulnerability or other security-related bug in this project, please read our [security policy]
+and report the bug to our Security Team 🛡️ Thank you!
 
 e-mail contact: security@arduino.cc
 
-## License
-
-Copyright (c) 2015-2021 Arduino LLC. All right reserved.
-
-This library is free software; you can redistribute it and/or modify it under the terms of the GNU Lesser General Public
-License as published by the Free Software Foundation; either version 2.1 of the License, or (at your option) any later
-version.
-
-This library is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied
-warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
-details.
-
-You should have received a copy of the GNU Lesser General Public License along with this library; if not, write to the
-Free Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA
-
+[tests-badge]: https://github.com/Arduino/arduino-fwuploader/workflows/test/badge.svg
+[docs-badge]: https://github.com/Arduino/arduino-fwuploader/workflows/publish-docs/badge.svg
```

Comment:
```suggestion
[tests-badge]: https://github.com/arduino/arduino-fwuploader/actions/workflows/test.yaml/badge.svg
[docs-badge]: https://github.com/arduino/arduino-fwuploader/actions/workflows/publish-docs.yaml/badge.svg
```
I like the other URLs, but I'm not sure how well supported it is, since GitHub's own "Create Status Badge" feature points to these ones.